PythonRegression/tests/features/steps/api_test_steps.py

```diff
@@ -124,23 +124,93 @@ def compare_thread_return(step,apiCall):
     logger.info('Responses match')
 
 
-@step(r'GTTA is called (\d+) times on "([^"]+)"')
-def spam_call_gtta(step,numTests,node):
-    apiCall = 'getTransactionsToApprove' 
-    world.config['apiCall'] = apiCall
+#TODO: Break these functions down into utilities
```

Comment:
In IRI's code I decided to not have TODOs in final PRs, but instead to write ISSUE # and open an issue.
I think I want to adopt the same practice here. GIthub should be used for issue tracking not the code.

You can use TODOs while working of course to make sure your PRs are completed.